build.rs

```diff
@@ -1,12 +1,11 @@
-#![deny(warnings)]
+// #![deny(warnings)]
```

Comment:
Seems like this might have been left over by accident?

Reply:
Yep, thanks for catching that!